docs/library/typography/link.md

```diff
@@ -21,6 +22,12 @@ You can also provide local links to other pages in your project without writing
 rx.link("Example", href="/docs/library",)
 ```
 
+```md alert info
+# Redirect the user
```

Comment:
I think this alert belongs at the bottom of the link docs above style, this spot is a little random for it. Also, title should be more descriptive maybe something like  "Redirecting in within  State"

Reply:
moved